src/components/Badges/BadgeStack.tsx

```diff
@@ -12,6 +12,7 @@ function BadgeStack({ panel, toggleFunction, hide }: Props) {
   if (hide) return null;
   return (
     <Box
+      onClick={() => toggleFunction(panel)}
```

Comment:
It was a nit in a previous PR to remove inline functions, can you move these to a dedicated usecallback function?

Reply:
No longer needed since the component structure changed